Userland/Utilities/ping.cpp

```diff
@@ -157,7 +157,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         struct timeval tv_send;
         gettimeofday(&tv_send, nullptr);
 
-        if (count && total_pings == count)
+        if (count.has_value() && total_pings == count.value())
```

Comment:
So this will loop forever if `count` is larger than `UINT32_MAX` because then `total_pings` can never reach it (it will rollover to 0).

You can leave the max limit on line 70 as `UINT32_MAX` to prevent this, or update all appropriate types to `size_t`.

Reply:
Ah, good catch. We'll just keep it at `UINT32_MAX` for now.